src/components/Calender/Calendar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { DateDisplay } from './DateDisplay';
+import { DayNameRow } from './DayNameRow';
+import { setFirstDay, compareDates } from '../../services/Calendar/helper_functions';
+import { months } from '../../services/Calendar/models';
+import { calendarCardDimensions } from '../../shared/constants';
+import { Text, Layout, Card } from '@ui-kitten/components';
+import { TaskData } from '../../shared/models';
+
+interface Props {
+    displayedDate: Date;
+    tasks?: TaskData[];
+    onDateTilePress?: (date, data) => void;
+    reducerType: string;
+}
+
+export const Calendar = React.memo(
+    (props: Props) => {
+        const { tasks, displayedDate } = props;
```

Comment:
couldn't you have done this on line 18?

Reply:
can you destructure reducerType and onDateTilePress as well?

Reply:
yarp